pkg/controller/container-runtime-config/container_runtime_config_bootstrap.go

```diff
@@ -32,7 +34,12 @@ func RunContainerRuntimeBootstrap(templateDir string, crconfigs []*mcfgv1.Contai
 			}
 			role := pool.Name
 			// Generate the original ContainerRuntimeConfig
-			originalStorageIgn, _, _, err := generateOriginalContainerRuntimeConfigs(templateDir, controllerConfig, role)
+			originalStorageIgn, _, _, err := generateOriginalContainerRuntimeConfigs(
+				templateDir,
+				controllerConfig,
+				role,
+				osimagestream.TryGetOSImageStreamFromPoolListByPoolName(osImageStream, mcpPools, pool.Name),
```

Comment:
question: why does the non-base MC generation code (for example, the containerruntimeconfig here) need the osimagestream info?